index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Apple Clone</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="container">
+        <ul class="list-inline">
+          <li>
+            <a href="http://www.apple.com" class="logo-brand image">
+              <img src="images/apple.svg" alt="logo" />
+            </a>
+          </li>
+          <li><a href="http://store.apple.com/">Store</a></li>
+          <li><a href="http://www.apple.com/mac/">Mac</a></li>
+          <li><a href="http://www.apple.com/ipod/">iPod</a></li>
+          <li>
+            <a href="http://www.apple.com/iphone">iPhone</a>
+          </li>
+          <li><a href="http://www.apple.com/ipad/">iPad</a></li>
+          <li>
+            <a href="https://www.apple.com/itunes/">iTunes</a>
+          </li>
+          <li>
+            <a href="http://www.apple.com/support/">Support</a>
+          </li>
+          <li>
+            <form action="#"  class="grab-search">
+              <i class="fas fa-search"></i>
+              <input type="text" name="search" />
+            </form>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="welcome-section">
+        <div class="container">
+          <div class="welcome-text">
+            <h1>What will your verse be?</h1>
+            <h2>
+              Everyone has something to share. See how people are using iPad in
+              their unique ways to add to the world’s story.
+            </h2>
+            <h3>Watch the “Your Verse” film and explore the stories</h3>
+          </div>
+        </div>
+      </section>
+      <section class="promo-ads">
+        <div class="row">
+          <div class="col-25">
```

Comment:
Each of this four articles should be a link. Use the `a` tag to wrap the content and make it clickable.